Employee

Kura33 opened the pull request:
merge models employee.

Kura33 committed:
test

Kura33 committed:
test

Kura33 committed:
test

Kura33 committed:
rebase

Kura33 committed:
Merge branch 'event' of https://github.com/Kura33/ProjectJavaCesi.git into event

Kura33 committed:
Merge branch 'event' of https://github.com/Kura33/ProjectJavaCesi.git into event

Kura33 committed:
implement EventType class + refacto.

Kura33 committed:
creation branch employee.
add class modele Employee and EmployeeDepartment.

Kura33 committed:
update modele employee and incident

Kura33 reviewed (commented):
ok

mcpeyran commented on src/main/java/com/shield/projectJavaCesi/entity/employee/Employee.java:
```diff
@@ -0,0 +1,104 @@
+	private String password;
+
+	public enum Contract {
+		CDI("CDI"), CDD("CDD"), ALTERNANCE("Alternance"), STAGE("Stage"), NOT_CONCERNED("N/C");
```
Dans Incident les clés des variables de énum sont en anglais, les valeurs en français. Ici les deux sont en français. On devrait choisir l'un ou l'autre, celui que tu veux. Merci : )

Zekemaz replied:
Je suis d'accord, 

- CDI : OPEN_ENDED_CONTRACT
- CDD : FIXED_TERM_CONTRACT
- ALTERNANCE : APPRENTICESHIP
- STAGE : INTERNSHIP
- NOT_CONCERNED est bon :)

Kura33 committed:
save end day.
identification to bdd for entity's module table
link table's entity module
Creation Interface repository
creation Service Event
creation method save, get, update and delete for entity's service.
work in progress updateIncident

Zekemaz reviewed (changes requested):
Je suis d'accord avec le commentaire de Marie-Cécile

CDI : OPEN_ENDED_CONTRACT
CDD : FIXED_TERM_CONTRACT
ALTERNANCE : APPRENTICESHIP
STAGE : INTERNSHIP
NOT_CONCERNED est bon :)
